apps/web/src/components/Search.tsx

```diff
@@ -2,23 +2,23 @@ import React, { ChangeEvent, KeyboardEvent, useEffect, useState } from 'react';
 import searchIcon from '@assets/img/search.svg';
 import clearIcon from '@assets/img/x_clear.svg';
 import barIcon from '@assets/img/bar.svg';
-import { ApplicantRO } from '@ien/common';
 
 interface SearchProps {
   onChange: (name: string) => void;
-  search: (name: string, limit: number) => Promise<ApplicantRO[]>;
-  onSelect: (id: string) => void;
+  search: (name: string, limit: number) => Promise<any[]>;
+  onSelect?: (id: string) => void;
   keyword?: string;
+  showDropdown?: boolean;
 }
 
 const QUERY_LIMIT = 5; // limit the number of search results
 const QUERY_DELAY = 300; // to reduce number of api calls
 const FOCUS_OUT_DELAY = 300; // to make redirection to detail page working
 
 export const Search = (props: SearchProps) => {
-  const { keyword, search, onChange, onSelect } = props;
+  const { keyword, search, onChange, onSelect, showDropdown } = props;
```

Comment:
I am not sure if it is good to extend this component's functionality to the case we don't need dropdown list of search results.

Reply:
what do you recommend?